### PR TITLE
Project pizza, week 1

### PR DESCRIPTION
**Pull request by emmydieden**

### CODE REVIEW

File: code/script.js [hunk 1]
```diff
@@ -1,19 +1,164 @@
...
+} else if (foodChoice === "2") {
+  selectedFood = "Pasta"
+} else if (foodChoice === "3") {
+  selectedFood = "Salad"
+} else {
+  alert("Invalid choice. Please select valid number.");
+  process.exit(1)
+}
+
+alert(`You've chosen ${selectedFood}!`)
```
Comment (AnnaRobertsson) [inline code/script.js:31]: Nice job!

File: code/script.js [hunk 2]
```diff
@@ -1,19 +1,164 @@
...
 // Your code goes here
+const age = prompt(`Is this food for a child or an adult? Type your age`);
+let price = (age >= 12) ? 150 : 75;
+let ageType = (age >= 12 ? "adult" : "child")
+
 
 // Step 5 - Order confirmation
 // Your code goes here
+
+const confirm = prompt(`One ${ageType} sized ${selectedSubtypeName} will be prepared for you. That'll be ${price} SEK. Are you sure you want to order this?
```
Comment (AnnaRobertsson) [inline code/script.js:150]: Looks nice!

**Review by AnnaRobertsson — CHANGES_REQUESTED** [review]
First of all - Great job! I can tell you put a lot of effort into it. Please take a look at my comments to see my input.

_Overall thoughts:_
I like the way you present the project in your Read me file, maybe a bit of that can be added on the pull request "info text" as well? I appreciate and applaud you for redoing step 3 when you felt it could be improved, really good job with that! As far as I can tell everything works perfectly fine from the exception of one thing I noticed which you can read more about in my comment under step 4.

I saw there was a script2.js and I tried to look for changes compared to script.js but I didn't see any when I did a quick look. Did I miss anything?

In conclusion - Good job, be proud of what you have accomplished!
